service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -58,7 +58,8 @@ public String mapEncounterToEhrComposition(Encounter encounter) {
             .components(components)
             .code(buildCode(encounter))
             .displayName(buildDisplayName(encounter))
-            .originalText(buildOriginalText(encounter));
+            .originalText(buildOriginalText(encounter))
+            .authorTime("20100113151332");
```

Comment:
hard-coded time ?

Reply:
Believe authorTime is calculated from lines 70 to 75 here
So this hardcoded line can be removed